### DISCUSSION TITLE
이거 이클립스에선 안 돌아가요 ㅠㅠ

### DISCUSSION THREAD

**Discussion by ugamezeRepo** [cloudchamb3r/Animalls#63] [Q&A] [answered]
-- DELIVERY_ADDRESS
BEGIN
 FOR i IN 1..50 LOOP
  INSERT INTO DELIVERY_ADDRESS 
   (delivery_id, postal_address, address, address_detail) 
   VALUES (LPAD(delivery_address_seq.NEXTVAL, 8, '0'), LPAD(i, 5, '0'), 'dum' || i, 'dummy' || i);
 END LOOP;
END;

**Comment by cloudchamb3r** [comment, marked as answer]
이클립스 기본 sql editor에서는 PL/SQL 을 지원하지 않는걸로 보입니다
DBEAVER 플러그인 설치후 간단한 세팅 하신다면 PL/SQL 쿼리도 실행이 가능합니다

**Reply by ugamezeRepo** [reply]
그럼 더미 데이터 만드는 방법이 세 가지겠네여..
1. 저대로 코드 쓰고 플러그인 설치하라고 한다.
2. sqlplus 실행해서 쓰라고 한다.
3. 풀어서 노가다로 만든다...
뭘로 할까여 ㅋㅋ

**Reply by cloudchamb3r** [reply]
플러그인만 깔면 플러그인 쓰는게 편하긴 하더라고요
그런데 이건 다른분들 .project 파일도 변경할 가능성이 있어서 조금 생각을 해봐야 될거같아요
pl/sql 관련파일은 따로 분리하시는게 나을거같기도합니다. 그냥 사람들이 알아서 실행할수있게요
